File: Going to the cinema.dart
int movie(int card, int ticket, double perc) {
  var counter = 0;
  num dticket = ticket;
  num sum = card;
  while (ticket * counter <= sum.ceil()) {
    counter++;
    sum += dticket *= perc;
  }
  return counter;
  
}

File: Going to the cinema/Going to the cinema.swift
func movie(card: Double, ticket: Double, perc: Double) -> Int {
    var cnt = 0
    var currentCard = card
    while Double(ticket) * Double(cnt) <= ceil(currentCard) {
        cnt += 1
        currentCard += ticket * pow(perc, Double(cnt))
    }
    return cnt
}
